Libraries/LibGfx/ICOLoader.cpp

```diff
@@ -279,15 +298,15 @@ static bool load_ico_bitmap(ICOLoadingContext& context, size_t index)
         desc.bitmap = png_decoder.bitmap();
         if (!desc.bitmap) {
 #ifdef ICO_DEBUG
-            printf("load_ico_bitmap: failed to load PNG endcoded image index: %lu\n", index);
+            printf("load_ico_bitmap: failed to load PNG encoded image index: %d\n", index);
```

Comment:
Please squash fixes like this into the previous commit instead of having them here. :)